tasks.py

```diff
@@ -121,6 +121,18 @@ def parse_spec_for_filters(_):
             handle.write(_filter + "\n")
 
 
+@task
+def get_markdown_spec(ctx):
+    print("Attempting to run pandoc...")
+    ctx.run(
+        "pandoc "
+        "-f rst -t markdown_strict "
+        "--wrap=preserve --columns=50000 "
```

Comment:
```suggestion
        "-f rst "
        "-t markdown_strict "
        "--wrap=preserve "
        "--columns=50000 "
```
Just a suggestion, not necessary.